### DISCUSSION TITLE
igmpproxy not working after 2.5 upgrade

### DISCUSSION THREAD

**Discussion by molney2** [fabianishere/udm-iptv#244] [Q&A]
Tried many different methods to get igmpproxy to work include improxy but never works.  Appears to add ip to the routing table but then still freezes.

I have tried igmpproxy both within udm-iptv and by itself with similar results.  TV freezes as it moves from unicast to multicast.

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 0 (dev iptv mac 90:9d:7d:48:f3:e1)
WAN DHCP: false (options "")
WAN Ranges: 239.192.0.0/16 239.195.0.0/16
LAN Interfaces: br3
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
4: eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    inet 108.82.254.233/22 scope global dynamic eth8
       valid_lft 3124sec preferred_lft 3124sec
108.82.252.0/22 proto kernel scope link src 108.82.254.233
=== Service Logs ===
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[send_igmp_mld_query@155]: igh_len  = 8
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[group_source_specific_timer_handler@786]: REMOVE
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[imp_check_timer@171]: continue............
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[main@653]: Data arrive
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[mcast_recv_igmp@742]: src addr = 192.168.10.4
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: received interface  29
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[imp_input_report_v1v2@282]: group address 239.192.7.15
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[main@653]: Data arrive
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: DEBUG[mcast_recv_igmp@742]: src addr = 192.168.10.2
Mar 18 17:25:31 Dream-Machine-Pro udm-iptvd[30322]: received interface  29

**Comment by FThuijs** [comment]
I had the same problem. Tried a lot of settings. But what finally helpt me was uninstalling udm-iptv and then installing it again. I even could use igmpproxy (read in an other post that it was not working anymore.
By the way: I'm working with an UDM-PRO firmware version 2.5.17.

**Reply by Edu-ST** [reply]
Hi @FThuijs  I tried reinstall udm-iptv after update to 2.5.17, but I have got this message
unifi-os: command not found , on 2.5.17

Could you support me please !!!  how did you run the script without access   unifi-os shell ?
thank you !!!

**Comment by FThuijs** [comment]
Hi Eduardo,What I did was, from the unifi-os shell, the adviced installation procedure again: so starting with the curl command and so on.Good luck!FrankOp 7 mei 2023 om 05:11 heeft Eduardo ***@***.***> het volgende geschreven:﻿
Hi @FThuijs  I tried reinstall udm-iptv after update to 2.5.17, but I have got this message
unifi-os: command not found , on 2.5.17
Could you support me please !!!  how did you run the script without access   unifi-os shell ?
thank you !!!

**Reply by Edu-ST** [reply]
Thank you for answer....
 
I did, directly, but I have got this error...
Did you get it also and keep the proccess?  I tried run configure command but nothing...   maybe I am making wrong..

root@UDM-Pro:~# sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
Downloading packages...
Installing packages...
Ign:1 http://security.debian.org stretch/updates InRelease
Ign:2 http://deb.debian.org/debian stretch InRelease
Ign:3 http://security.debian.org stretch/updates Release
Ign:4 http://deb.debian.org/debian stretch-updates InRelease
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:6 http://deb.debian.org/debian stretch-backports InRelease
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:8 http://deb.debian.org/debian stretch Release
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:10 http://deb.debian.org/debian stretch-updates Release
Ign:11 http://deb.debian.org/debian stretch-backports Release
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Err:7 http://security.debian.org stretch/updates/contrib arm64 Packages
  404  Not Found [IP: 199.232.114.132 80]
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Err:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
  404  Not Found [IP: 199.232.114.132 80]
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Err:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
  404  Not Found [IP: 199.232.114.132 80]
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Err:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
  **404  Not Found [IP: 199.232.114.132 80]**
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Hit:29 https://apt-release-candidate.artifacts.ui.com stretch InRelease
Hit:30 https://apt-beta.artifacts.ui.com stretch InRelease
Hit:31 https://apt.artifacts.ui.com stretch InRelease
Reading package lists...
W: The repository 'http://security.debian.org stretch/updates Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-updates Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-backports Release' does not have a Release file.
E: Failed to fetch http://security.debian.org/dists/stretch/updates/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-updates/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-backports/main/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: **Some index files failed to download. They have been ignored, or old ones used instead.**
root@UDM-Pro:~#
root@UDM-Pro:~# udm-iptv configure
-bash: **udm-iptv: command not found**

**Comment by FThuijs** [comment]
Hi Eduardo,I see you get a lot of Debian update messages. As far as I can remember I did not get those. After the curl command I got directly the configure menu.Maybe a reboot of your UDM Pro first? There is also version 3.0 available for UDM Pro. Maybe better to upgrade first to that version if you want to do so anyway.Tell me if you did succeed. Good luck!FrankOp 7 mei 2023 om 15:37 heeft Eduardo ***@***.***> het volgende geschreven:﻿
Thank you for answer....
I did, directly, but I have got this error...
Did you get it also and keep the proccess?  I tried run configure command but nothing...   maybe I am making wrong..
***@***.***:# sh -c "$(curl https://raw.githubusercontent.com/fabianishere/udm-iptv/master/install.sh -sSf)"
Downloading packages...
Installing packages...
Ign:1 http://security.debian.org stretch/updates InRelease
Ign:2 http://deb.debian.org/debian stretch InRelease
Ign:3 http://security.debian.org stretch/updates Release
Ign:4 http://deb.debian.org/debian stretch-updates InRelease
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:6 http://deb.debian.org/debian stretch-backports InRelease
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:8 http://deb.debian.org/debian stretch Release
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:10 http://deb.debian.org/debian stretch-updates Release
Ign:11 http://deb.debian.org/debian stretch-backports Release
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:7 http://security.debian.org stretch/updates/contrib arm64 Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:5 http://security.debian.org stretch/updates/main all Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Err:7 http://security.debian.org stretch/updates/contrib arm64 Packages
404  Not Found [IP: 199.232.114.132 80]
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:9 http://security.debian.org stretch/updates/non-free arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:15 http://security.debian.org stretch/updates/contrib all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:16 http://security.debian.org stretch/updates/non-free all Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:18 http://security.debian.org stretch/updates/main arm64 Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Ign:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Ign:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Ign:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Err:12 http://deb.debian.org/debian stretch/contrib arm64 Packages
404  Not Found [IP: 199.232.114.132 80]
Ign:13 http://deb.debian.org/debian stretch/main arm64 Packages
Ign:14 http://deb.debian.org/debian stretch/main all Packages
Ign:17 http://deb.debian.org/debian stretch/contrib all Packages
Ign:19 http://deb.debian.org/debian stretch/non-free arm64 Packages
Ign:20 http://deb.debian.org/debian stretch/non-free all Packages
Err:21 http://deb.debian.org/debian stretch-updates/contrib arm64 Packages
404  Not Found [IP: 199.232.114.132 80]
Ign:22 http://deb.debian.org/debian stretch-updates/non-free all Packages
Ign:23 http://deb.debian.org/debian stretch-updates/non-free arm64 Packages
Ign:24 http://deb.debian.org/debian stretch-updates/contrib all Packages
Ign:25 http://deb.debian.org/debian stretch-updates/main all Packages
Ign:26 http://deb.debian.org/debian stretch-updates/main arm64 Packages
Err:27 http://deb.debian.org/debian stretch-backports/main arm64 Packages
404  Not Found [IP: 199.232.114.132 80]
Ign:28 http://deb.debian.org/debian stretch-backports/main all Packages
Hit:29 https://apt-release-candidate.artifacts.ui.com stretch InRelease
Hit:30 https://apt-beta.artifacts.ui.com stretch InRelease
Hit:31 https://apt.artifacts.ui.com stretch InRelease
Reading package lists...
W: The repository 'http://security.debian.org stretch/updates Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-updates Release' does not have a Release file.
W: The repository 'http://deb.debian.org/debian stretch-backports Release' does not have a Release file.
E: Failed to fetch http://security.debian.org/dists/stretch/updates/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-updates/contrib/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Failed to fetch http://deb.debian.org/debian/dists/stretch-backports/main/binary-arm64/Packages  404  Not Found [IP: 199.232.114.132 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.
***@***.***:#
***@***.***:~# udm-iptv configure
-bash: udm-iptv: command not found

**Reply by Edu-ST** [reply]
@FThuijs   thank you so much buddy !!!
Your tip and suggest were perfect!!   I got run the configure menu normally!
Really you help me a lot !!!

Have a great weekend...
